### PR TITLE
feat(Rating): add Rating component

### PR DESCRIPTION
**Pull request by Unic0arn**

### CODE REVIEW

File: src/components/rating/rating.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+        const UiKit = require('../../');
```
Comment (JacobBlomgren) [inline src/components/rating/rating.md]: I'm in the process of upgrading styleguidist, and from now on code examples need to be wrapped in code blocks: 

\```js
\<Code />
\```

Just wanted to spread that knowledge 😃

File: test/components/rating/rating.test.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+import { Icon, mockClasses } from '../../../src';
+import { Component as Rating, styles } from '../../../src/components/rating/rating';
+
+const classes = mockClasses(styles);
+
+const getComponent = props => {
```
Comment (JacobBlomgren) [inline test/components/rating/rating.test.js]: `renderComponent` if I can be really picky

File: src/components/rating/rating.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+const Rating = ({ icon: RenderIcon = Icon.Star, size, rating, maxRating, classes }) => (
+  <div className={classes.rating}>
+    {[...Array(maxRating)].map((_, index) => {
+      const classNames = cn({
+        [classes.active]: index < rating,
+        [classes.inactive]: index >= rating,
+        [classes.ratingIcon]: true,
+      });
+      return (
+        <RenderIcon
```
Comment (JacobBlomgren) [inline src/components/rating/rating.jsx]: Rename to `Icon` maybe? `RenderIcon` sounds a bit weird.

Comment (Unic0arn) [reply src/components/rating/rating.jsx]: Collides with the imported Icon

**Pull request merged**